indico/modules/events/editing/operations.py

```diff
@@ -204,8 +214,25 @@ def create_submitter_revision(prev_revision, user, files):
     return new_revision
 
 
+@memoize
+def _is_request_changes_with_files(revision):
+    if not revision or len(revision.editable.revisions) < 2:
+        return False
+    previous_revision = revision.editable.revisions[-2]
+    return (revision != previous_revision and
```

Comment:
I think the logic here deserves a comment about what in here makes it as "request changes with files" and not a normal "request changes".